### DISCUSSION TITLE
Failed to read archive

### DISCUSSION THREAD

**Discussion by Ibrahim77077** [wstyres/Zebra#2071] [Q&A]
When I install any package I found this error 
Failed to read archive 
Which means Application folder in library 
And when I check in Filza I didn't find it there. 
What is wrong and how can I fix it?
Thank you
![image](https://github.com/zbrateam/Zebra/assets/63207581/c8beb0f4-0c55-423d-ad00-a7c232516ba0)

**Comment by iAdam1n** [comment]
This is not a Zebra error.

**Reply by Ibrahim77077** [reply]
I found this error after I cleaned the device by iCleaner

**Reply by Ibrahim77077** [reply]
So how can I fix it?

**Reply by iAdam1n** [reply]
I don't know, better to ask on rjailbreak Discord - it's nothing to do with Zebra.

**Reply by Ibrahim77077** [reply]
> I don't know, better to ask on rjailbreak Discord - it's nothing to do with Zebra.

Thank you
